processor/transform/tables-to-jsx.ts

```diff
@@ -39,8 +41,8 @@ const visitor = (table: Table, index: number, parent: Parents) => {
       return EXIT;
     }
 
-    visit(cell, 'text', (text: Text) => {
-      if (text.value.match(/\n/)) {
+    visit(cell, isLiteral, (node: Literal) => {
+      if (node.value.match(/\n/)) {
```

Comment:
What is a "literal" node anyways?

Reply:
Anything with a text value. I think that's `text`, `inlineCode`, `code`, `html`, `yaml`